docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -0,0 +1,105 @@
+[role="xpack"]
+[[dfa-regression]]
+== {regression-cap}
+
+{reganalysis-cap} is a {ml} process for estimating the relationships among 
+different fields in your data, then making further predictions based on these 
+relationships.
+
+For example, suppose we are interested in finding the relationship between 
+apartment size and monthly rent in a city. Our imaginary dataset consists of 
+three data points:
+
+|===
+| Size (m2) | Monthly rent 
+| 44        | 1600
+| 24        | 1055
+| 63        | 2300
+|===
+
+After the model determines the relationship between the
+(apartment size) and the {depvar} (rent), it can make predictions such as the 
+monthly rent of a hundred square meter-size apartment.
+
+This is a simple example. Usually {regression} problems are multi-dimensional, 
+so the relationships that {reganalysis} tries to find are between multiple 
+fields. To extend our example, a more complex 
+{reganalysis} could take into account further {feature-vars} like the location 
+of the apartment in the city, on which floor it is, and whether the apartment 
+has a riverside view or not and so on.
+
+
+[discrete]
+[[dfa-regression-features]]
+=== {feature-vars-cap}
+
+When you perform {reganalysis}, you must identify a subset of fields that you 
+want to use to create a model for predicting other fields. We refer to these 
+fields as _feature variables_ and _dependent variables_, respectively.
+{feature-vars-cap} are the values that the {depvar} value depends on. If one or 
+more of the {feature-vars} changes, the {depvar} value also changes. There are 
+three different types of {feature-vars} that you can use with our {regression} 
+algorithm:
+
+* Numerical. In our example, the size of the apartment was a 
+  numerical {feature-var}.
+* Categorical. A variable that can have one value from a set of values. The 
+  value set has a fixed and limited number of possible items. In the example, 
+  the location of the apartment in the city (borough) is a categorical variable.
+* Boolean. The riverside view in the example is a boolean value because an 
+  apartment either has a riverside view or doesn't have one.
+Arrays are not supported.
+
+
+[discrete]
+[[dfa-regression-supervised]]
+=== Training the {regression} algorithm
+
+{regression-cap} is a supervised machine learning process, which means that you 
+need to supply a labeled training dataset that has some {feature-vars} and a 
+{depvar}. The {regression} algorithm learns the relationships between the 
+features and the {depvar}. Once you've trained the model on your training 
```

Comment:
I think this need to either be chanced to "feature and dependent variables", "feature and dependent variable" or "feature variable and dependent variables":
```suggestion
feature and the {depvar}. Once you've trained the model on your training 
```